types/xsd-schema-validator/tsconfig.json

```diff
@@ -19,4 +19,4 @@
         "index.d.ts",
         "xsd-schema-validator-tests.ts"
     ]
-}
```

Comment:
Ugh, looks like some unrelated files from a previous commit got in. I'm going to exclude those when I resubmit (since the CI build failed).